src/main/webapp/resources/js/apis/metadata/metadata-templates.js

```diff
@@ -13,3 +13,28 @@ export function getProjectMetadataTemplates(projectId) {
     .get(`${BASE_URL}?projectId=${projectId}`)
     .then(({ data }) => data);
 }
+
+/**
+ * Get details about a specific metadata template
+ * @param {number} templateId Identifier for a metadata template
+ * @returns {Promise<AxiosResponse<any>>}
+ */
+export async function getMetadataTemplateDetails({ templateId }) {
+  return await axios
+    .get(setBaseUrl(`${BASE_URL}/${templateId}`))
```

Comment:
Set this setBaseUrl(`${BASE_URL}/${templateId}`) at the top of the file as it is used it multiple functions

Reply:
I remove the `setBaseUrl` in this specific function call as that was already at the top of the page.  I cannot move the entire `${BASE_URL}/${templateId}` up since `templateId` is not known until runtime.

Updated in 683f10a

Reply:
Awesome thanks